debezium-platform/postgresql-kafka-example/README.md

```diff
@@ -0,0 +1,147 @@
+
+Using the debezium-platfrom-stage(UI) for seting up our data pipeline 
+---
+Now once you have running platfrom-stage(UI), we will create a data pipeline and all its resources i.e source, destination and transform(as per need) thru it. You will see different side navigation option to configure them.
+
+For this demo, see the connection properties you can use for each connector type as illustrated below:
+
+### Source
+ #### PostgreSQL
+
+ ```shell
+  {
+  "name": "test-source",
+  "description": "postgreSQL database",
+  "type": "io.debezium.connector.postgresql.PostgresConnector",
+  "schema": "schema123",
+  "vaults": [],
+  "config": {
+    "topic.prefix": "inventory",
+    "database.port": 5432,
+    "database.user": "debezium",
+    "database.dbname": "debezium",
+    "database.hostname": "postgresql",
+    "database.password": "debezium",
+    "schema.include.list": "inventory"
+  }
+}
+
+ ```
+
+![PostgreSQL Connnector](./resources/source.png)
+
+
+### Destination
+ ```shell
+ {
+  "name": "test-destination",
+  "description": "Kafka destination",
+  "type": "kafka",
+  "schema": "schema123",
+  "vaults": [],
+  "config": {
+    "producer.key.serializer": "org.apache.kafka.common.serialization.StringSerializer",
+    "producer.value.serializer": "org.apache.kafka.common.serialization.StringSerializer",
+    "producer.bootstrap.servers": "dbz-kafka-kafka-bootstrap.debezium-platform:9092"
+  }
+}
+
+ ```
+
+ ![Kafka Connnector](./resources/destination.png)
+
+### Transform
+
+**Transform class**: o.debezium.transforms.ExtractNewRecordState       
+**Transform name**: Debezium marker      
+**Description**: Extract Debezium payload  
+**Adds the specified fields to the header if they exist**: db,table  
+**Adds the specified field(s) to the message if they exist.**: op  
+**Predicate type**: org.apache.kafka.connect.transforms.predicates.TopicNameMatches    
+**Pattern**: inventory.inventory.products
+
+ ```shell
+ {
+  "config": {
+    "add.fields": "op",
+    "add.headers": "db,table"
+  },
+  "description": "Extract Debezium payload",
+  "name": "Debezium marker",
+  "predicate": {
+    "config": {
+      "pattern": "inventory.inventory.products"
+    },
+    "negate": false,
+    "type": "org.apache.kafka.connect.transforms.predicates.TopicNameMatches"
+  },
+  "schema": "string",
+  "type": "io.debezium.transforms.ExtractNewRecordState",
+  "vaults": []
+}
+
+ ```
+
+ ![ExtractNewRecordState](./resources/transform.png)
+
+### Pipeline
+The use of [Operator Lifecycle Manager](https://olm.operatorframework.io/) allows you to configure the scope of namespaces watched by the operator from a single namespace to the entire cluster. The process below will install the operator into the `operators` namespace -- which is by default intended for cluster-wide operators. 
```

Comment:
I suppose this paragraph could be removed since we are installing it with `Helm`